resources/ru/cemu.po

```diff
@@ -459,19 +460,22 @@ msgstr "Выберите режим эмуляции CPU"
 msgid "Thread quantum"
 msgstr "Thread quantum"
 
+# thread slice runtime?
```

Comment:
This is the maximum amount of time given to a thread to perform its work before switching context (the original strings explains this technical concept very badly).